### ISSUE TITLE
20230713.819 do not work 

### ISSUE THREAD

**Issue by ghost** [nod-ai/SHARK#1749]
I checkout the 20230713.819 but it doesn't work, it 's block to "Loading flatbuffer at [...]_vulkan.vmfb as a mmapped file"

**Comment by NeedsMoar** [comment]
Odd, 819 is the last version that _does_ work for me.  After that everything fails with compilation errors until 872, but _that_ version has exactly the same freeze you're getting with 819.  I'm using the Windows packaged EXE though, not the source tree.  I'm wondering if one of the new versions of iree-runtime or iree-compiler is breaking things, since those are only specified as >= version requirements in setup.py but the version in the executable package is frozen to whatever the current was when the package was built.

**Comment by powderluv** [comment]
Yeah we are waiting on a driver update before the next stable release. The mmap loading thing will work if you wait 10min.
